scripts/evergreen/release/update_release.py

```diff
@@ -31,6 +31,9 @@ def update_release_json():
     newest_operator_version = data["mongodbOperator"]
     update_operator_related_versions(data, newest_operator_version)
 
+    if not update_latest_om_agent_mapping(data, newest_om_version):
```

Comment:
`update_latest_om_agent_mapping` doesn't need to return anything and in case of issues just raise an Exception with proper message. 

Currently the errors are printed using `debug` level which is inadequate for errors.